package/Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM alpine
-RUN apk update && apk add -u --no-cache git curl unzip tar tini bash nfs-utils && \
```

Comment:
Why remove nfs-utils? Is nfs utility built-in for bci-base image?

Reply:
https://github.com/harvester/harvester/pull/1909
It should be safe to remove nfs-utils since Harvester doesn't use it.

Reply:
Actually, the `nfs-utils` is still needed as we just move the dependency library to https://github.com/longhorn/backupstore, and the nfs mount within the container will still require to use this library.

Reply:
Thanks, I add the corresponding package `nfs-client`: https://github.com/harvester/harvester/compare/75cc5c6fd7b1b6ac25b0256106a24532d7bec41a..47630964c440f93cb9148a6044246bb7eb129b64